docs/specifications/tests/Basic-TP/basic04.md

```diff
@@ -3,6 +3,20 @@
 ## Test case identifier
 **BASIC04**
 
+
+## Table of contents
+
+* [Objective](#Objective)
+* [Scope](#Scope)
+* [Inputs](#Inputs)
+* [Summary](#Summary)
+* [Test procedure](#Test-procedure)
+* [Outcome(s)](#Outcomes)
+* [Special procedural requirements](#Special-procedural-requirements)
+* [Intercase dependencies](#Intercase-dependencies)
+* [Terminology](#terminology)
+
+
 ## Objective
 
 Many test cases will query the name servers in the delegation or the
```

Comment:
SOA record or both. Reporting problem is crucial -> SOA record, or both. Reporting problems is crucial

Reply:
Fixed.